randomActivityGen.py

```diff
@@ -127,10 +130,21 @@ def main():
     stats = ET.parse(args["--stat-file"])
     verify_stats(stats)
 
-    logging.info("Writing Perlin noise to population and industry")
+    max_display_size = int(args["--display.size"])
+
     centre = find_city_centre(net) if args["--centre.pos"] == "auto" else tuple(
         map(int, args["--centre.pos"].split(",")))
 
+    # If display-only, load stat-file as input and exit after rendering
+    if args["--display-only"]:
+        # Try the output-file first, as, if given, it contains a computed statistics file, otherwise try the input
+        stats = ET.parse(args["--output-file"] if not None else args["--stat-file"])
```

Comment:
`not None` is always true and the program crashes if `--output-file` is not given. You either want to do
```python
stats = ET.parse(args["--stat-file"] if args["--output-file"] is None else args["--output-file"])
```
or use the shorthand
```python
stats = ET.parse(args["--output-file"] or args["--stat-file"])
```

Reply:
Of course, that's a brainfart. Fixed in 1cc2964